### PR TITLE
add runcam split support

### PR DESCRIPTION
**Pull request by azolyoung**
Hi there,

Here is azol, I'm adding runcam split control to iNav FC, and for compatibility with older versions that not support rcsplit, I've change the apiVersion to 1.27.0, so only the  iNav FC firmwares which apiVersion >= 1.27.0 can support rcsplit. Hope that is correctly place.

Thank you!

### CODE REVIEW

File: src/main/msp/msp_protocol.h [hunk 1]
```diff
@@ -60,7 +60,7 @@
 #define MSP_PROTOCOL_VERSION                0
 
 #define API_VERSION_MAJOR                   1 // increment when major changes are made
-#define API_VERSION_MINOR                   26 // increment when any change is made, reset to zero when major changes are released after changing API_VERSION_MAJOR
+#define API_VERSION_MINOR                   27 // increment when any change is made, reset to zero when major changes are released after changing API_VERSION_MAJOR
```
Comment (digitalentity) [inline src/main/msp/msp_protocol.h]: @azolyoung Actual MSP API didn't change, no need to increase it. You should probably check for firmware version in the Configurator.

Comment (azolyoung) [reply src/main/msp/msp_protocol.h]: @digitalentity 
Hi, May I know what is the next FC version? I'm going to use FC version as the condition for rcsplit support

Comment (digitalentity) [reply src/main/msp/msp_protocol.h]: @azolyoung you can check for version 1.7.3 and above. Current stable release is 1.7.2 so this won't affect anything but development branch.

Comment (azolyoung) [reply src/main/msp/msp_protocol.h]: @digitalentity 
OK, I get it! 
Thank you!

File: src/main/target/common.h [hunk 2]
```diff
@@ -40,6 +40,7 @@
 #define TELEMETRY
 #define TELEMETRY_LTM
 #define TELEMETRY_FRSKY
+#define USE_RCSPLIT
```
Comment (digitalentity) [inline src/main/target/common.h]: Please move this to `FLASH_SIZE > 128` section. It won't fit into F1-based targets.

Comment (azolyoung) [reply src/main/target/common.h]: OK!

**Comment by digitalentity** [comment]
@azolyoung did you test this? I'm eager to merge it.

**Comment by azolyoung** [comment]
@digitalentity 

Hi, I've double check it, it's work on my split with the latest split firmware.

**Comment by maxxb1** [comment]
Hi guys can’t get the split(version 1) to work with omnibus f3 board. I’m running inav 1.8 and latest split firmware. 
Tried different uarts, switched rx and tx, nothing worked ;(

**Comment by fiam** [comment]
@maxxb1 1.8 only supports RC split firmware 1.0. You need to use the development branch or wait for the next release.

**Comment by maxxb1** [comment]
@fiam thx for the quick answer. I thought according to the messages above and the other threads that it was already integrated in 1.8 
If it overcomes me where can I find the development branch? ( sorry for the questions, I’m new here and don’t how how it works on github )

**Pull request merged**